x-pack/plugins/fleet/public/applications/fleet/sections/agent_policy/edit_package_policy_page/components/index.ts

```diff
@@ -0,0 +1,8 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export { UpgradeStatusCallout } from './upgrade';
```

Comment:
I started a refacto to write more unit test for the client side but noticed this is already cover by a unit test, let me know if you think it make sense to keep that refacto (just extracting a component out of the huge edit_package_policy index file)

Reply:
Seems fine to me to have this component extracted.